legend-depot-store-metrics/src/main/java/org/finos/legend/depot/store/metrics/services/QueryMetricsHandler.java

```diff
@@ -18,19 +18,28 @@
 import org.finos.legend.depot.domain.project.ProjectVersion;
 import org.finos.legend.depot.store.admin.api.metrics.QueryMetricsStore;
 import org.finos.legend.depot.store.admin.domain.metrics.VersionQueryMetric;
+import org.slf4j.Logger;
 
 import javax.inject.Inject;
 import java.util.Comparator;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.Optional;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class QueryMetricsHandler
 {
     public static final Comparator<VersionQueryMetric> MOST_RECENTLY_QUERIED = (o1, o2) -> o2.getLastQueryTime().compareTo(o1.getLastQueryTime());
+    public static final Comparator<VersionQueryMetric> MOST_RECENTLY_QUERIED_TIME = (o1, o2) ->
```

Comment:
this is identical to above functionality wise they do the same, please remove one ..before reads better than compare